### PR TITLE
Implement better handling of long files and fixes

### PR DESCRIPTION
**Pull request by KyleFromNVIDIA**

### CODE REVIEW

File: src/rapids_pre_commit_hooks/lint.py [hunk 1]
```diff
@@ -117,31 +120,52 @@ def print_warnings(self, fix_applied=False):
...
+                        print("note: suggested fix")
                 print()
 
     def print_highlighted_code(self, pos, replacement=""):
         line_index = self.line_for_pos(pos[0])
         line_pos = self.lines[line_index]
         left = pos[0] - line_pos[0]
 
+        if replacement:
+            replacement = f" {replacement}"
```
Comment (bdice) [inline src/rapids_pre_commit_hooks/lint.py:153]: Why add a space here? I think it's more confusing than the previous output.

**Review by bdice — APPROVED** [review]
One small question, but otherwise LGTM.

File: src/rapids_pre_commit_hooks/lint.py (hunk 1 above)
Comment (bdice) [reply src/rapids_pre_commit_hooks/lint.py:153]: An alternative would be to write a diff/patch style output with - and +. That would also make the replacement line shorter, without all the ~~~~~~~~~~ characters.

Comment (bdice) [reply src/rapids_pre_commit_hooks/lint.py:153]: I’m fine with whatever you decide here.

Comment (KyleFromNVIDIA) [reply src/rapids_pre_commit_hooks/lint.py:153]: Diff-style output with `+` and `-` makes sense, but it wouldn't highlight the parts *within* the line that need to be changed (unless we did something fancy with terminal color output, which I'm open to doing.)

The `~~~` format was inspired by clang's warning output.

Comment (KyleFromNVIDIA) [reply src/rapids_pre_commit_hooks/lint.py:153]: I just checked out `rich` and agree that it look good for formatting the output. I'll use it to show a nicer diff.

Comment (KyleFromNVIDIA) [reply src/rapids_pre_commit_hooks/lint.py]: I've redone the output using `rich` and created a diff-like output for the replacements.

**Pull request merged**